cloud-run-single/1-apps/main.tf

```diff
@@ -32,6 +32,8 @@ module "cloud_run" {
       subnet  = local.subnet_id
       tags    = var.cloud_run_configs.vpc_access_tags
     }
+    gpu_zonal_redundancy_disabled = var.cloud_run_configs.gpu_zonal_redundancy_disabled
```

Comment:
can you please put these lines at the beginning of the revision block before the other dictionary?